### PR TITLE
DVX-5064: Adds odin to create RDS instances

### PR DESCRIPTION
**Pull request by ifosch**

### CODE REVIEW

File: mcc/odin/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (miguelbernadi) [inline mcc/odin/LICENSE]: This LICENSE file shouldn't be here, right?

**Comment by towanda** [comment]
test failing...

**Comment by ifosch** [comment]
@miguelbernadi @towanda fixed

File: mcc/odin/cmd/create.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+var size int64
+
+// createCmd represents the create command
+var createCmd = &cobra.Command{
+	Use:   "create [flags] identifier",
+	Short: "Creates a database",
+	Long:  `Creates a database, into RDS.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 1 {
+			log.Fatal("You should include instance identifier")
```
Comment (miguelbernadi) [inline mcc/odin/cmd/create.go]: This doesn't cover well the case where there's more than 1 argument. Minor thing, though.

Comment (ifosch) [reply mcc/odin/cmd/create.go]: Well, right now, it only expects one single argument, being the instance name.

File: mcc/odin/odin/odin.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+	instanceName string,
+	instanceType string,
+	masterUser string,
+	masterUserPassword string,
+	size int64,
+	svc rdsiface.RDSAPI,
+) (result string, err error) {
+	params := &rds.CreateDBInstanceInput{
+		AllocatedStorage:     &size,
+		DBInstanceClass:      aws.String(instanceType),
```
Comment (miguelbernadi) [inline mcc/odin/odin/odin.go]: These being variables passed can be captured directly with `&instanceType` and `&instanceName`

File: mcc/odin/odin/odin.go [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+		DBInstanceIdentifier: aws.String(instanceName),
+		Engine:               aws.String("postgres"),
+		EngineVersion:        aws.String("9.4.11"),
+		DBSecurityGroups: []*string{
+			aws.String("default"),
+		},
+		MasterUserPassword: aws.String(masterUserPassword),
```
Comment (miguelbernadi) [inline mcc/odin/odin/odin.go]: Same as above, but for `&masterUserPassword` and `&masterUser`

File: mcc/odin/odin/odin.go [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+		MasterUsername:     aws.String(masterUser),
+		Tags: []*rds.Tag{
+			{
+				Key:   aws.String("Name"),
+				Value: aws.String(instanceName),
```
Comment (miguelbernadi) [inline mcc/odin/odin/odin.go]: Same as above

File: mcc/odin/odin/odin.go (hunk 3 above)
Comment (ifosch) [reply mcc/odin/odin/odin.go]: Fixed now

File: mcc/odin/odin/odin.go (hunk 4 above)
Comment (ifosch) [reply mcc/odin/odin/odin.go]: Fixed now

File: mcc/odin/odin/odin.go (hunk 5 above)
Comment (ifosch) [reply mcc/odin/odin/odin.go]: Fixed now

File: mcc/odin/cmd/create.go (hunk 2 above)
Comment (miguelbernadi) [reply mcc/odin/cmd/create.go]: I mean message says there's a missing parameter but it can fail because of too many arguments. Misleading message.

File: mcc/odin/odin/odin.go [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+	for *instance.DBInstanceStatus != "available" {
+		res2, err2 := svc.DescribeDBInstances(&rds.DescribeDBInstancesInput{
+			DBInstanceIdentifier: instance.DBInstanceIdentifier,
+		})
+		if err2 != nil {
+			err = err2
+			return
+		}
+		instance = *res2.DBInstances[0]
+		time.Sleep(duration)
```
Comment (miguelbernadi) [inline mcc/odin/odin/odin.go]: Why is this here? Maybe a comment explaining it would be useful.

Comment (ifosch) [reply mcc/odin/odin/odin.go]: I put it to avoid Rate throttling at AWS API...
I'm adding a simple comment in the code.

**Comment by miguelbernadi** [comment]
+1

File: mcc/odin/cmd/create.go (hunk 2 above)
Comment (ifosch) [reply mcc/odin/cmd/create.go]: Maybe now it's clearer... I also added some more test cases for empty and lacking parameters.

**Comment by towanda** [comment]
+1
